data-prepper-expression/src/main/java/org/opensearch/dataprepper/expression/CoercionException.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+public class CoercionException extends Exception {
```

Comment:
This is a public class, please add Javadocs.

Reply:
That is a mis. I should have made it package private